tests/vmi_multus_test.go

```diff
@@ -606,8 +606,7 @@ var _ = Describe("[Serial]SRIOV", func() {
 
 		tests.BeforeTestCleanup()
 		// Check if the hardware supports SRIOV
-		sriovcheck := checkSriovEnabled(virtClient, sriovResourceName)
-		if !sriovcheck {
+		if err := validateSRIOVSetup(virtClient, sriovResourceName, 1); err != nil {
 			Skip("Sriov is not enabled in this environment. Skip these tests using - export FUNC_TEST_ARGS='--ginkgo.skip=SRIOV'")
```

Comment:
Maybe make the `err` part of the Skip message?

Reply:
> Looks good to me in general, two nits inline.

Hi thanks for reviewing this,
the changes suggest mentions are actually from #4741 this PR rebased on.
I prefer not to change these until they merged, we could also those on #4741